includes/talks/functions.php

```diff
@@ -1191,179 +1191,6 @@ function wct_talks_update_talk() {
 	exit();
 }
 
-/** Sticky talks **************************************************************/
-
-/**
- * Gets the sticky talks
- *
- * @package WordCamp Talks
- * @subpackage talks/functions
- *
- * @since 1.0.0
- *
- * @return array the list of IDs of sticked to front talks
- */
-function wct_talks_get_stickies() {
-	$sticky_talks = get_option( 'sticky_talks', array() );
-
-	/**
-	 * @param  array $sticky_talks the talks sticked to front archive page
-	 */
-	return apply_filters( 'wct_talks_get_stickies', $sticky_talks );
-}
-
-/**
- * Edit WP_Query posts to append sticky talks
- *
- * Simply a "copy paste" of how WordPress deals with sticky posts
- *
- * @package WordCamp Talks
- * @subpackage talks/functions
- *
- * @since 1.0.0
- *
- * @param  array    $posts The array of retrieved posts.
- * @param  WP_Query The WP_Query instance
- * @return array the posts with stickies if some are found
- */
-function wct_talks_stick_talks( $posts = array(), $wp_query = null ) {
```

Comment:
As you're removing this function, to avoid a notice error, we also need to remove the filter that is added to 'the_posts' in `includes/core/filters.php`.

```diff
diff --git includes/core/filters.php includes/core/filters.php
index 5c87f73..7681abe 100644
--- includes/core/filters.php
+++ includes/core/filters.php
@@ -34,9 +34,6 @@ add_filter( 'protected_title_format', 'wct_talks_protected_title_prefix', 10, 2
 // Order by rates count
 add_filter( 'posts_clauses', 'wct_set_rates_count_orderby', 10, 2 );
 
-// Sticky Talks
-add_filter( 'the_posts', 'wct_talks_stick_talks', 10, 2 );
-
 // Filter comment author urls
 add_filter( 'comments_array',               'wct_comments_array',               11, 2 );
 add_filter( 'comments_template_query_args', 'wct_comments_template_query_args', 10, 1 );